### ISSUE TITLE
If some one fork this repository this should automatically work under username 

### ISSUE THREAD

**Issue by charithsoori** [99xt/99xt.github.io#22]
What is left for forked user
- Rename project
- Set analytics id

**Comment by chamathpali** [comment]
Lets setup a config.json and get the custom settings like 

1. Logo URL
2. Org/Profile Name
3. GITHUB User name (org name/ personal)
4. Google Analytics Code snippet

**Comment by PedroCSSilva** [comment]
Can you give me some extra tips? I would like to help.

**Comment by chamathpali** [comment]
yeah. we can have a seperate json file including the custom options like the above mentioned,

Logo URL
Org/Profile Name
GITHUB User name (org name/ personal)
Google Analytics Code snippet

then we can load them in js, this will make the configuration part easy for a user who just fork the repo and hoping to use it

**Comment by charithsoori** [comment]
@PedroCSSilva Did you start to look into this ?

**Comment by PedroCSSilva** [comment]
I'll look at it during the weekend, sorry guys.
